aten/src/ATen/test/apply_test.cpp

```diff
@@ -1,121 +1,119 @@
-#define CATCH_CONFIG_MAIN
-#include "catch_utils.hpp"
+#include "gtest/gtest.h"
 
 #include "cuda.h"
 #include "cuda_runtime.h"
 
 #include "ATen/cuda/detail/TensorInfo.cuh"
 
 /*
-Tests related to tensor indexing and applying operations. 
+Tests related to tensor indexing and applying operations.
 */
 #ifndef _WIN32
 
-CATCH_TEST_CASE("2D Contiguous", "Collapses a 2D contiguous tensor to 1D contiguous") {
-    int sizes[] = {4, 4};
-    int strides[] = {4, 1};
-    ::at::cuda::detail::TensorInfo<void, int> ti{nullptr, 2, sizes, strides};
-    ti.collapseDims();
-    CATCH_REQUIRE(ti.dims == 1);
-    CATCH_REQUIRE(ti.sizes[0] == (4 * 4));
+TEST(apply_test, Contiguous_2D) {
```

Comment:
Please don't discard the test name (the second string). We're just losing information here that may be very descriptive and important. You should instead put it as a comment in the first line of the test